Dockerfile

```diff
@@ -22,13 +21,18 @@ RUN apk --update add \
     rm /var/cache/apk/* && \
     ln -s /usr/bin/php7 /usr/bin/php
 
+COPY composer.json /usr/src/app/composer.json
+COPY composer.lock /usr/src/app/composer.lock
+
 RUN apk --update add curl && \
     curl -sS https://getcomposer.org/installer | php && \
     ./composer.phar install && \
     apk del curl && \
     rm /usr/src/app/composer.phar \
        /var/cache/apk/*
 
+COPY bin/build-content /usr/src/app/bin/build-content
```

Comment:
Hm I like that idea, but how do you write that? Trying

```
COPY composer.*|bin ./
```

But that ... does not seem to work 😅